Packet++/src/S7CommLayer.cpp

```diff
@@ -0,0 +1,117 @@
+#include "EndianPortable.h"
+
+#include "S7CommLayer.h"
+#include "TcpLayer.h"
+#include <iostream>
+
+#include <cstring>
+#include <sstream>
+
+namespace pcpp
+{
+
+	S7CommLayer::S7CommLayer(uint8_t msgType, uint16_t pduRef, uint16_t paramLength, uint16_t dataLength,
```

Comment:
We don't set `m_Parameter` anywhere in this constructor. I think we can set is to `nullptr`